README.rst

```diff
@@ -2,39 +2,32 @@
 plone.behavior
 ==============
 
-This package provides optional support for "behaviors". A behavior is
-a re-usable aspect of an object that can be enabled or disabled without
-changing the component registry.
-
-A behavior is described by an interface, and has metadata such as a title and
-a description. The dotted name of the interface is the unique name for the
-behavior, from which the metadata can be looked up. When the behavior is
-enabled for an object, you will be able to adapt the object to the interface.
+This package provides support for **behaviors**.
+A behavior is a re-usable aspect of an object that can be enabled or disabled without changing the component registry.
+
+A behavior is described by an interface, and has metadata such as a title and a description.
+The dotted name of the interface is the unique name for the behavior, from which the metadata can be looked up.
+When the behavior is enabled for an object, you will be able to adapt the object to the interface.
 In some cases, the interface can be used as a marker interface as well.
 
-As an example, let's say that your application needs to support object-level
-locking, and that this can be modeled via an adapter, but you want to leave
-it until runtime to determine whether locking is enabled for a particular
-object. You could then register locking as a behavior.
+As an example, let's say that your application needs to support object-level locking.
+This can be modeled via an adapter, but you want to leave it until runtime to determine whether locking is enabled for a particular object.
+You could then register locking as a behavior.
 
 Requirements
 ------------
 
-This package comes with support for registering behaviors and factories. It
-does not, however, implement the policy for determining what behaviors are
-enabled on a particular object at a particular time. That decision is deferred
-to an `IBehaviorAssignable` adapter, which you must implement.
+This package comes with support for registering behaviors and factories.
+It does not, however, implement the policy for determining what behaviors are enabled on a particular object at a particular time.
+That decision is deferred to an ``IBehaviorAssignable`` adapter, which must be implemented.
+I.e. the ``plone.dexterity`` package implements this.
```

Comment:
Not sure "i.e." is appropriate here. Maybe "plone.dexterity implements this for you"?